module/purpose/Attack/src/Attack.cpp

```diff
@@ -35,6 +37,7 @@ namespace module::purpose {
                 log<DEBUG>("We have the ball or it is free, walk to the goal!");
                 // Try to walk to the ball and align towards opponents goal
                 emit<Task>(std::make_unique<WalkToKickBall>(), 3);
+                emit<Task>(std::make_unique<KickTo>(), 4);  // kick the ball towards the goal
```

Comment:
I'd probably make this general to any attack case - ie use it in the Tackle as well. Then we have a better chance of knocking the ball out from the opponent's feet. If you're not sure about it and want it tested first, can skip for now and can check it later.

Reply:
Is that what you meant?